src/benchmark.sh

```diff
@@ -19,13 +20,19 @@ PPCSR_PLOT_FILE=$(mktemp gnuplot.pXXX)
 PPCSR_PDF_PLOT_FILE=ppcsr_strong_scaling_plot
 
 # Define experiment parameters
-REPETITIONS=3
+REPETITIONS=2
```

Comment:
Sorry, just realised I left it at 2 instead of 3 (for quicker testing). Will return it back to 3 in the next PR that will improve few bits in the script.